src/components/ArbeidsgiverPeriodeTabell.less

```diff
@@ -52,49 +46,40 @@
 
   &--søke-gruppe {
     display: grid;
-    grid-template-columns: auto auto;
+    grid-template-columns: auto auto 100px;
```

Comment:
Hvor mye support har vi for grid-template-columns?

Reply:
Ah, right. Må lave det for IE support :/

Reply:
IE support bliver ny PR